frontend/src/app/main/ui/components/search_bar.cljs

```diff
@@ -52,15 +41,20 @@
     [:span {:class (stl/css-case :search-box true
                                  :has-children (some? children))}
      children
-     [:div {:class (dm/str input-class " " (stl/css :search-input-wrapper))}
-      icon
+     [:div {:class (dm/str class " " (stl/css :search-input-wrapper))}
```

Comment:
Minor enhancement: 

The class can be defined as `:class  [class (stl/css :search-input-wrapper)]` instead of explicit concat